tests/VivViewerLayerBase.spec.js

```diff
@@ -18,7 +18,7 @@ test('VivViewerLayerBase', t => {
     assert: t.ok,
     sampleProps: {
       sliderValues: [0, 10],
-      colorValues: [[0, 1, 1], [], [], [], [], []],
+      colorValues: [0, 1, 1],
```

Comment:
Is this now just a single rgb? Consider renaming.

Reply:
Ah no, the data is flattened.  There is a comment about this in the code.